group_vars/all/.gitignore

```diff
@@ -1 +1,2 @@
 dynamic.yml
+reverseproxy.yml
```

Comment:
Do we have an reverseproxy.yml.example? Otherwise I think adding the vars which are in reverseproxy to dynamic are also fine and just leave them commented out so the proxy is only setup when the vars are defined.

Reply:
no, we didn't have an example for that file, and at the end it was added to the role.

Reply:
I removed that addition